README.md

```diff
@@ -31,32 +31,29 @@ Static assets are the files (javascript, css, images) that are generated from a
 
 All of the leading application frameworks ([Angular CLI](https://github.com/angular/angular-cli/wiki/stories-application-environments), [Create React App](https://cli.vuejs.org/guide/mode-and-env.html#using-env-variables-in-client-side-code), [Ember CLI](https://ember-cli.com/user-guide/#Environments), [Vue CLI 3](https://cli.vuejs.org/guide/mode-and-env.html#using-env-variables-in-client-side-code)) recommend defining environment _values_ at _compile time_. This practice requires that the static assets are generated for each environment and regenerated for any change to an environment.
 
-_Immutable Web Applications_ reference environment _variables_ that are  defined on the global scope and that are set to values in the `index.html` that are unique to the deployment:
+_Immutable Web Applications_ reference environment _variables_ that are defined on the global scope and either referenced directly on the `window` object or through an injected service that is a wrapper for the environment variables:
```

Comment:
I understand this sentence, but honestly it's a little hard to read.

> _Immutable Web Applications_ reference environment _variables_ that are defined on the global scope and reference one of two ways:
> - Directly from the window object
> - Through an injected service that wraps the environment variables

Reply:
agreed and updated.